### PR TITLE
add test for fill_value Tensor and refine the doc test=develop

### PR DESCRIPTION
**Pull request by wangchaochaohu**
Refine the doc and add support for value Tensor

image

image
image

image

### CODE REVIEW

File: python/paddle/tensor/creation.py [hunk 1]
```diff
@@ -353,36 +353,49 @@ def full(shape,
                 The data type is ``int32`` or ``int64`` . If ``shape`` is a list or tuple,
                 the elements of it should be integers or Tensors with shape [1].
                 If ``shape`` is an Variable, it should be an 1-D Tensor .
-        value(float): The constant value used to initialize the Tensor to be created.
+        fill_value(float16|float32|float64|int32|int64|Variable): The constant value used to initialize
```
Comment (zhaoyuchen2018) [inline python/paddle/tensor/creation.py]: bool?

File: python/paddle/tensor/creation.py [hunk 2]
```diff
@@ -353,36 +353,49 @@ def full(shape,
...
         out(Variable, optional): Optional output which can be any created 
             Variable that meets the requirements to store the result of operation.
             if out is None, a new Varibale will be create to store the result.
         dtype(np.dtype|core.VarDesc.VarType|str, optional): Data type of the output tensor
             which can be float16, float32, float64, int32, int64, if dytpe is `None`, the data
             type of created tensor is `float32`
-        device(str, optional): This parameter specifies that the Tensor is created 
-            on the GPU or CPU.
+        device(str, optional): Which device to run the operator. The :attr:`device` must be
+            None, 'cpu', 'gpu'. If :attr:`device` is None, it will be choose the device that the user set in
```
Comment (zhaoyuchen2018) [inline python/paddle/tensor/creation.py]: maybe it will choose?

File: python/paddle/tensor/creation.py (hunk 1 above)
Comment (wangchaochaohu) [reply python/paddle/tensor/creation.py]: Done Thanks

File: python/paddle/tensor/creation.py (hunk 2 above)
Comment (wangchaochaohu) [reply python/paddle/tensor/creation.py]: Done Thanks

**Review by zhaoyuchen2018 — DISMISSED** [review]
LGTM

**Review by liym27 — DISMISSED** [review]
LGTM

**Review by liym27 — APPROVED** [review]

**Pull request merged**